.vscode/c_cpp_properties.json

```diff
@@ -5,7 +5,8 @@
             "includePath": [
                 "${workspaceFolder}/**",
                 "${workspaceFolder}/../../libraries/hwlib/library",
-                "${workspaceFolder}/../../programs/bmptk/targets/cortex/atmel"
+                "${workspaceFolder}/../../programs/bmptk/targets/cortex/atmel",
+                "${workspaceFolder}/../../libraries/Catch2/include/*"
```

Comment:
this shouldn't be part of this PR, best to add this feature in a PR to the [build repo](https://github.com/R2D2-2019/R2D2-build) or from a separate branch within robot arm